docs/cri/config.md

```diff
@@ -421,6 +421,16 @@ version = 2
     # * Containerd imgcrypt: https://github.com/containerd/imgcrypt
     key_model = "node"
 
+  # 'plugins."io.containerd.grpc.v1.cri".image_verification' contains config related
+  # to handling runtime verification of container images.
+  [plugins."io.containerd.grpc.v1.cri".image_verification]
+    # address specifies address of the TTRPC plugin handing image verification
+    # requests.
```

Comment:
We use gRPC for most plugins

Reply:
We can also consider exec()-ing a verifier binary, as in stream processor binaries and binary loggers.

(We don’t support binary loggers for CRI, but there is on-going discussion about that)

Reply:
I think TTRPC or gRPC would yield lower verification latencies for some plugin implementations, since long-lived resources (e.g. network connections or caches) could be reused across multiple verification requests. TTRPC was recommended in a previous meeting with @mikebrow, but from my perspective as a user of this feature, either protocol meets my requirements.